build.gradle

```diff
@@ -0,0 +1,156 @@
+buildscript {
+  ext {
+    springBootVersion = '1.5.8.RELEASE'
+  }
+  repositories {
+    mavenCentral()
```

Comment:
please remove mavenCentral as we use jcenter

Reply:
changed